src/main/java/seedu/address/logic/commands/EditMeetingCommand.java

```diff
@@ -72,15 +73,17 @@ public CommandResult execute(Model model) throws CommandException {
         Meeting meetingToEdit = lastShownList.get(index.getZeroBased());
         Meeting editedMeeting = createEditedMeeting(meetingToEdit, editMeetingDescriptor);
 
-        if (!meetingToEdit.equals(editedMeeting) && model.hasSpecificMeeting(editedMeeting)) {
+        if (model.hasMeeting(editedMeeting)) {
```

Comment:
to be clear why this is necessary for it to work, the premise of noconflictmeetinglist is to disallow conflicting meetings. by checking for the same exact meeting, this doesn't help prevent conflicts.